students/Sahlberg/Lesson6/UnitTest.py

```diff
@@ -0,0 +1,92 @@
+"""Ian Sahlberg
+Python210
+2/13/2019
+Unit Testing"""
+
+
+import unittest as u
+import Mailroom3 as m
+from Mailroom3 import amount_append
+from Mailroom3 import donate
+from Mailroom3 import print_report
+from Mailroom3 import quit
+
+
+class tests(u.TestCase):
```

Comment:
`class MailroomTest(unittest.TestCase):`

* camel case class name
* singular names for class names
* don't use alias imports